libs/components/router/src/lib/modules/href/href.directive.ts

```diff
@@ -93,88 +111,90 @@ export class SkyHrefDirective {
     altKey: boolean,
     metaKey: boolean
   ): boolean {
-    if (!this._route || !this._route.userHasAccess) {
+    if (!this.#_route || !this.#_route.userHasAccess) {
```

Comment:
I know this isn't your fault, but we shouldn't be accessing `#_` properties outside of the getter/setters.

Reply:
That was just a misnamed property - it isn't for a getter/setter so it should just be `#route`. Fixed.